drivers/mtd/nand/raw/gpmi-nand/gpmi-nand.c

```diff
@@ -90,7 +90,7 @@ static int gpmi_reset_block(void __iomem *reset_addr, bool just_enable)
 	if (!just_enable) {
 		/* set SFTRST to reset the block */
 		writel(MODULE_SFTRST, reset_addr + MXS_SET_ADDR);
-		udelay(1);
+		udelay(10); /* Increased from 1 for PORTF-1119 */
```

Comment:
So 10µs is the actual tested delay value that reduced timeout event to 1:1500?

Reply:
I am testing this now.
It is not clear to me why this code uses both udelay and timer.
It appears that the udelay is some minimum and the timer is some type of fine-tuning on top of the minimum.
The actual results are that either the readl succeeds on the first attempt, or the timeout is zero. There are no instances in between.